src/bit_set.rs

```diff
@@ -15,26 +15,21 @@ use std::slice;
 #[derive(Default,Debug)]
 pub struct Bitset {
     head: usize,
-    tail: Option<Box<Vec<usize>>>,
+    tail: Option<Vec<usize>>,
 }
 
 fn bits_per_word() -> usize {
     usize::max_value().count_ones() as usize
 }
 
-#[inline(never)]
-fn clone_slow(arg: &Box<Vec<usize>>) -> Box<Vec<usize>> {
-    arg.clone()
-}
-
 impl Clone for Bitset {
     #[inline]
     fn clone(&self) -> Bitset {
```

Comment:
Inline the fast path, force deinline the slow path.  Yes, I measured this.  Note that `tail` will *never* be a Some when verifying set.mm on a 64-bit machine, it only comes up when a function uses more than 64 variables